frontend/report.20200409.022812.282598.0.001.json

```diff
@@ -0,0 +1,612 @@
+
```

Comment:
What is this? Some debugging report?

Reply:
yes, these are created when webpack build goes out of memory, My bad didn't notice this.